apt.html

```diff
@@ -65,6 +65,19 @@ <h2>{% t apt.apt.repo %}</h2>
 
             <pre><code># apt-get install bitcoinxt</code></pre>
 
+	    <p>{% t apt.apt.locatepassword %}</p>
+
+	    <pre><code># cat /etc/bitcoinxt/bitcoin.conf</code></pre>
+
+	    <p>{% t apt.apt.run %}</p>
+
+	    <pre><code># bitcoinxt-cli -rpcuser=bitcoinxt -rpcpassword=AutoGeneratedPasswordFromBitcoinDotConfGoesHere getblockchaininfo</code></pre>
```

Comment:
I think it's better to replace `-rpcuser=bitcoinxt -rpcpassword=autogeneratedpassword` with `-conf=/etc/bitcoinxt/bitcoin.conf`. 

It's shorter and the operator doesn't have to look up the contents of the config file.

If you do, then you can also remove the section about locating password.